reminders/views.py

```diff
@@ -1,3 +1,29 @@
-# from django.shortcuts import render
+from django.utils import timezone
+from django.http import JsonResponse
+from reminders.models import Notification
+from django.contrib.auth.decorators import login_required
 
-# Create your views here.
+
+@login_required(login_url="login")
+def get_notification(request):
+    user = request.user
+
+    user_notifications = list(
+        Notification.objects.filter(
+            participant_id__user_id=user,
+            seen_time__isnull=True
+        ).values('id', 'message')
+    )
+
+    return JsonResponse(user_notifications, safe=False)
+
+
+@login_required(login_url="login")
+def seen_notification(request, pk):
```

Comment:
can you please rename pk to somthing like notification_id?

Reply:
don. thanks 😄